glossary.yml

```diff
@@ -4418,13 +4418,19 @@
     term: "false negative"
     def: >
       Data points which are actually [true](#true) but incorrectly predicted as [false](#false).
+
+  pt:
+    term: "falso negativo"
+    def: >
+      Dados ou resultados que na realidade são [verdadeiro](#true) mas que são incorretamente considerados ou determinados como [falso](#false) pelo algoritmo.
```

Comment:
Suggest the following: Dados ou resultados que são realmente verdadeiros, mas são incorretamente previstos como falsos